### PR TITLE
finished string indices

### PR DESCRIPTION
**Pull request by lilvina**

### CODE REVIEW

File: spec/fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// import chai, { expect} from 'chai'
+// import chaiChange from 'chai-change'
+// import Fibonacci from '../src/fibonacci'
+
+// chai.use(chaiChange)
+
+// describe('Fibonacci', () => {
```
Comment (DianaVashti) [inline spec/fibonacci.js:7]: Did you comment out to not return tests that don't pass? You can achieve testing a specific file/function by using the describe.only method. Just remember to remove the .only when you move on to the next one.

File: spec/string-indices.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+chai.use(chaiChange)
+
+describe('StringIndices', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(StringIndices).to.be.a('function')
+  })
+
+  it('returns the string in the sentence found at the index', () => {
```
Comment (DianaVashti) [inline spec/string-indices.js:14]: a word within a string is a word, technically, and not it's own 'string'. I would rewrite this to say `returns the word at a specified index of a string`

File: spec/string-indices.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    expect(StringIndices('catch me outside how about that', 1)).to.equal('catch')
+  })
+
+  it('should return an empty string if you ask for the word at index 7', () => {
```
Comment (DianaVashti) [inline spec/string-indices.js:18]: I would be more specific to what you are testing for. You are not testing to make sure you only care about words 1-6 of any string. This is meant to test for asking for an index that does not exist. I would write this as `returns empty string if the index requested does not exist`

File: src/disemvowling.js [hunk 4]
```diff
@@ -1,38 +1,8 @@
...
-//     return str.replace(/[aeiou]/gi, "")
-//   }
-// }
-
-// export default Disemvoweling
-
 export default function Disemvoweling(str) {
   let removeSpace = str.replace(/\s/g, '')
   let removeVowel = removeSpace.replace(/[aeiou]/gi, '')
   return removeVowel
```
Comment (DianaVashti) [inline src/disemvowling.js:6]: What about capital vowels? Based on this implementation they would not be removed. You could use the `toLowerCase()` method after the `removeSpace` method. (e.g. `let lowerString = removeSpace.toLowerCase()` and then... `let removeVowel = lowerString.replace(/[aeiou]/gi,'')`)

File: src/string-indices.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+export default function StringIndices(input, index) {
+  let toArray = input.toString().split(' ')
```
Comment (DianaVashti) [inline src/string-indices.js:4]: I would name this `splitStringToArray` for clarity. Seeing a method that includes the `toString()` method that is called `toArray()` is a bit confusing if a person does not remember quickly that the `string.split()` method returns an array.

File: src/string-indices.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+  if(index >= toArray.length) {
```
Comment (DianaVashti) [inline src/string-indices.js:6]: this should be `if(index > toArray,length)` If it is `>=` then if the user requests the last index it returns an empty string. (e.g. as it's written `StringIndicies('cash me outside how bow dah', 6) -> ''` but it should `-> dah`

File: spec/fibonacci.js (hunk 1 above)
Comment (ai-santos) [reply spec/fibonacci.js:7]: yaay describe.only!

**Pull request merged**